### DISCUSSION TITLE
hi im not able to see all the nodes in prometheus.im attaching the screenshots below

### DISCUSSION THREAD

**Discussion by raju-b70** [learndevopsonline/learndevopsonline#1870] [Q&A]
![Screenshot (52)](https://github.com/learndevopsonline/learndevopsonline/assets/161826606/1579591b-07e2-404b-8fd9-a122d8828e07)
![Screenshot (53)](https://github.com/learndevopsonline/learndevopsonline/assets/161826606/1a5076ca-a798-427e-9b28-1b950bf63bfe)

**Comment by chaithanya1812** [comment]
Hi @raju-b70, join lab session tomorrow, It's needs debugging

**Reply by raju-b70** [reply]
okay chaitanya....i will thank you
